test/infrastructure/docker/main.go

```diff
@@ -43,6 +44,7 @@ func init() {
 	_ = scheme.AddToScheme(myscheme)
 	_ = infrav1.AddToScheme(myscheme)
 	_ = clusterv1.AddToScheme(myscheme)
+	_ = expv1.AddToScheme(myscheme)
```

Comment:
It seems a bit weird here to have the docker provider importing the experimental types when it isn't using them...